Cisco/cisco-secure-firewall/tests/test_106001.json

```diff
@@ -15,7 +15,8 @@
       "target": "network-traffic"
     },
     "destination": {
-      "address": "192.168.122.55",
+      "address": "LAN",
+      "domain": "LAN",
```

Comment:
According to my comprehension of the event, it seems to be the name of the interface.
So, I think that [`observer.egress.interface.name`](https://www.elastic.co/guide/en/ecs/8.16/ecs-interface.html#field-interface-name) is more suitable